libcasr/src/js.rs

```diff
@@ -0,0 +1,379 @@
+//! JS module implements `ParseStacktrace` and `Exception` traits for JS reports.
+use crate::error::{Error, Result};
+use crate::exception::Exception;
+use crate::execution_class::ExecutionClass;
+use crate::stacktrace::{ParseStacktrace, Stacktrace, StacktraceEntry};
+
+use regex::Regex;
+
+/// Structure provides an interface for parsing JS exception message.
+pub struct JsException;
```

Comment:
Add `CrashLineExt` impl

Reply:
And add update tests for it

Reply:
Don't need to impl `CrashLineExt`  
Added test